src/data/objects/domain.yaml

```diff
@@ -107,6 +107,25 @@ schema:
     filterable: true
     subtype: domain_type
     description: Controls the Domain type.
+  zonefile:
+    type: Object
+    description: >
+      Information about the rendered zone file.  This shows what is live right
+      now, and recent changes may take a minute to be reflected in this value.
```

Comment:
Is it an actual minute? I thought it could take up to fifteen minutes?

If it's more than an actual minute, we should specify with "may take up to ..." kind of message.